### DISCUSSION TITLE
Paperless: Change OCR language

### DISCUSSION THREAD

**Discussion by Spezialtrick** [tteck/Proxmox#1342] [Help Needed] [answered]
How can i change the OCR language of Paperless to German?

I've tried to change PAPERLESS_OCR_LANGUAGE=eng in /opt/paperless/paperless.conf to PAPERLESS_OCR_LANGUAGE=deu but after changing Paperless stopped importing documents through the consume directory.

**Comment by denes44** [comment, marked as answer]
You need to install the ocr language pack as well manually:
`apt install tesseract-ocr-deu`

**Reply by Spezialtrick** [reply]
Thank you very much! It's working now!
